src/python/twitter/common/reviewboard/reviewboard.py

```diff
@@ -103,7 +103,7 @@ def _add_auth_params(self, username, password):
     urllib2.install_opener(opener)
 
   def get_url(self, rb_id):
-    return '%s/r/%s' % (self.url, rb_id)
+    return '%sr/%s' % (self.url, rb_id)
```

Comment:
@slyphon Ack. 

@landism 
* You could fix that at sender, if suitable.
* or change to ```return self.url  + 'r/%s' % (rb_id)```  which I assume was the original intention.
* os.path.join() is okay, but is a hack in my opinion as rules of url joins are different from path joins.
* Or anything you see fit.
This is a minor issue or a change not *strictly required* IMO.